### PR TITLE
Rescue errors on reads

### PR DESCRIPTION
**Pull request by philipmuir**
Rescue errors on reads

@mechatama ping

### CODE REVIEW

File: lib/active_support/cache/cascade_store.rb [hunk 1]
```diff
@@ -77,7 +77,7 @@ def read_entry(key, options) # :nodoc:
         entry = nil
         synchronize do
           @stores.detect do |store|
-            entry = store.send(:read_entry, key, options)
+            entry = store.send(:read_entry, key, options) rescue nil
```
Comment (sgerrand) [inline lib/active_support/cache/cascade_store.rb:80]: Are you rescuing a specific error type? If so, can you be explicit?

Comment (qkos) [inline lib/active_support/cache/cascade_store.rb:80]: I believe this just wants to silently die and cascade to the next store.

**Comment by qkos** [comment]
:+1:  :hammer:

File: lib/active_support/cache/cascade_store.rb (hunk 1 above)
Comment (philipmuir) [inline lib/active_support/cache/cascade_store.rb:80]: this specifically - https://gist.github.com/philipmuir/72137da62bdeab2d66f6

but I think we should rescue all errors here and try the next cache provider

**Pull request merged**